lambda/test-indexbuild/src/main.ts

```diff
@@ -1,23 +1,21 @@
 import {DynamoDBClient} from "@aws-sdk/client-dynamodb";
-import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
-// eslint-disable-next-line import/no-unresolved -- this works from root tsconfig but eslint keeps complaining :-(
-import { type RecipeIndex, retrieveIndexData } from "@recipes-api/lib/recipes-data";
-import type { Handler } from "aws-lambda";
-import { StaticBucketName as Bucket } from "./config";
-
+import {PutObjectCommand, S3Client} from "@aws-sdk/client-s3";
+import type {Handler} from "aws-lambda";
+import {type RecipeIndex, retrieveIndexData} from "@recipes-api/lib/recipes-data";
+import {INDEX_JSON, V2_INDEX_JSON} from "../../recipes-responder/src/constants";
```

Comment:
This is the line that triggers a module-boundary warning.  As I mentioned before, if you want to share a value across multiple projects in this way you need to put it into the common library (which is fairly easy 😀 )